helper/geojsonify.js

```diff
@@ -62,7 +62,12 @@ function search( docs, params ){
   var geojson = GeoJSON.parse( geodata, { Point: ['lat', 'lng'] });
 
   // add bbox
-  geojson.bbox = extent( geojson ) || undefined;
+  try {
+    geojson.bbox = extent( geojson ) || undefined;
```

Comment:
yea, I think this is because `extent()` can return `null` which ends up in the JSON payload, I totally agree it doesn't feel right, will give it a rejig.